src/billing/List/BillingList.ts

```diff
@@ -86,17 +87,17 @@ export const BillingList = (props: BillingListProps) => {
     Utils.withBusyState(setIsAuthorizing)
   )(Auth.tryBillingScope);
 
-  const loadAccounts = _.flow(
-    reportErrorAndRethrow('Error loading billing accounts'),
-    Utils.withBusyState(setIsLoadingAccounts)
-  )(() => {
-    if (Auth.hasBillingScope()) {
-      return Ajax(signal)
-        .Billing.listAccounts()
-        .then(_.keyBy('accountName')) // @ts-ignore
-        .then(setBillingAccounts);
+  const loadAccounts = withHandlers(
```

Comment:
Was this change actually needed for this PR? Or are you just demonstrating this `withHandlers` function?